packages/api/src/schema/api-schema.yaml

```
@@ -1034,8 +1073,10 @@ components:
               description: Error message if the asset creation failed.
         name:
           type: string
-          description: |
-            The name of the asset. This is not necessarily the filename - it can be a
+          description: >
+            The name of the asset. This is not necessarily the filename - it can
+            be a
+
             custom name or title.
```

Potentially typo paragraph break here?
```suggestion
            be a custom name or title.
```